sticky_notes.py

```diff
@@ -43,6 +43,23 @@ def sender(x):
                 while len(notes) >= 1:
                     note = notes.pop()
                     self.delay_task(len(notes),
-                                    sender(format_note(nick, note)))
+                                    self.sender(format_note(nick, note), event))
                 self.notes[nick] = []
                 return format_note(nick, last_note)
+
+    @cmd
+    def list_stickynotes(self, event):
+        args = event.args
+        format_note = lambda x, y: "{0}. {1}".format(x, y)
+
+        if len(args) == 0:
+            return ' '.join(self.notes.keys())
```

Comment:
@JakubNvk I think it would help the overall UX if the bot said something like 'I have sticky notes for: nick1, nick2 ...'.